pavelib/servers.py

```diff
@@ -116,33 +156,68 @@ def celery(options):
 @needs('pavelib.prereqs.install_prereqs')
 @cmdopts([
     ("settings=", "s", "Django settings for both LMS and Studio"),
+    ("asset_settings=", "a", "Django settings for updating assets for both LMS and Studio (defaults to settings)"),
     ("worker_settings=", "w", "Celery worker Django settings"),
     ("fast", "f", "Skip updating assets"),
+    ("optimized", "f", "Run with optimized assets"),
```

Comment:
Actually, come to think of it, you probably don't even need a shortcut. It doesn't make sense to run just the install_prereqs task directly with a different asset settings file. Your call.